bouncer/static/scripts/redirect.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+/** The ID of the Chrome extension we want to talk to. */
+var EDITOR_EXTENSION_ID = 'oldbkmekfdjiffgkconlamcngmkioffd';
+
+/** Return the settings object that the server injected into the page. */
+function getSettings(document) {
+  return JSON.parse(
+    document.querySelector('script.js-bouncer-settings').textContent);
+}
+
+/** Navigate the browser to the given URL. */
+function navigateTo(url) {
+  window.location = url;
+}
+
+/** Navigate the browser to the requested annotation.
+ *
+ * If the browser is Chrome and our Chrome extension is installed then
+ * navigate to the annotation's direct link for the Chrome extension.
+ * If the Chrome extension isn't installed or the browser isn't Chrome then
+ * navigate to the annotation's Via direct link.
+ *
+ */
+function redirect(navigateTo_) {
```

Comment:
The convention we have for naming arguments that are functions is a `Fn` suffix, so `navigateFn`.